### DISCUSSION TITLE
Candlestick charts

### DISCUSSION THREAD

**Discussion by BaileySimrell** [tremorlabs/tremor-npm#256] [Ideas]
I think it would be great to have candlestick charts in the Tremor library to for creating visualizations of stock market data, etc.. and could be a great addition to the library. What do you all think?

**Comment by BaileySimrell** [comment]
For some added context, here's a good example of candlestick charts being used in [TradingView's `lightweight-charts` library](https://www.tradingview.com/lightweight-charts/): [https://codesandbox.io/s/9inkb](https://codesandbox.io/s/9inkb)

And here's another example implemented in [Plotly](https://github.com/plotly/plotly.js/): [https://plotly.com/javascript/candlestick-charts/](https://plotly.com/javascript/candlestick-charts/)

**Reply by mitrotasios** [reply]
Hey @BaileySimrell thanks a lot for the suggestion, I think that makes sense. We are generally planning to migrate to another, more customizable, chart library and make changes to our current charts api and we will prioritize that first. But we aim to expand our charts significantly in the long run, so we'll definitely consider candle charts as well! :)
